h/static/scripts/drafts.js

```diff
@@ -31,47 +39,53 @@ function DraftStore() {
    * unsaved drafts exist.
    */
   this.unsaved = function unsaved() {
-    return this._drafts.filter(function (draft) {
+    return this._drafts.filter(function(draft) {
       return !draft.model.id;
-    }).map(function (draft) {
+    }).map(function(draft) {
       return draft.model;
     });
-  }
+  };
 
   /** Retrieve the draft changes for an annotation. */
   this.get = function get(model) {
-    for (var i=0; i < this._drafts.length; i++) {
+    for (var i = 0; i < this._drafts.length; i++) {
       if (match(this._drafts[i], model)) {
-        return this._drafts[i].changes;
+        return {
+          private: this._drafts[i].private,
+          tags: this._drafts[i].tags,
+          text: this._drafts[i].text
```

Comment:
Really? I know this isn't JSON, but trailing commas are invalid in JSON, and I figured that for consistency with JSON we should do the same in JavaScript object literals